apps/fabric-website/src/pages/Controls/ButtonPage/ButtonPage.tsx

```diff
@@ -43,7 +47,46 @@ export class ButtonPage extends React.Component<
   public render() {
     const { areButtonsDisabled, areButtonsChecked } = this.state;
     const buttonPageProps = ButtonPageProps(areButtonsDisabled, areButtonsChecked);
-    return <ControlsAreaPage {...this.props} {...buttonPageProps[this.props.platform]} exampleKnobs={this.renderKnobs()} />;
+    return (
+      <ControlsAreaPage
+        {...this.props}
+        {...buttonPageProps[this.props.platform]}
+        exampleKnobs={this.renderKnobs()}
+        otherSections={this._otherSections(this.props.platform)}
+      />
+    );
+  }
+
+  private _otherSections(platform: Platforms): IPageSectionProps<Platforms>[] {
+    switch (platform) {
+      case 'ios':
+        return [
+          {
+            sectionName: 'Implementation',
+            editUrl: baseUrl + 'docs/ios/ButtonImplementation.md',
+            content: (
+              <Markdown>
```

Comment:
You can omit the `<Markdown>` wrapper--I recently made a change to the section control that automatically converts string `content` into markdown. (The reason you still see the `<Markdown>` wrapper in other places is just because I haven't taken the time yet to remove it everywhere.)